cluster/src/main/java/com/hartwig/pipeline/resource/ResourceFiles.java

```diff
@@ -110,6 +111,10 @@ default String gridssVirusRefGenomeFile() {
 
     String actionabilityDir();
 
+    default String cuppaRefData() {
+        return VmDirectories.resourcesPath(CUPPA);
```

Comment:
can we make this more consistent with the `of` method used by the other resources?